Timetabling.Tests/Objects/Constraints/TimeConstraints.Tests/ConstraintTeacherNotAvailableTimesTest.cs

```diff
@@ -1,67 +1,73 @@
-﻿using System.Collections.Generic;
-using System.Data.Entity;
-using System.Linq;
-using Moq;
-using NUnit.Framework;
-using Timetabling.DB;
-using Timetabling.Objects;
-using Timetabling.Objects.Constraints.TimeConstraints;
-
-namespace Timetabling.Tests.Objects.Constraints.TimeConstraints.Tests
-{
-    [TestFixture()]
-    internal class ConstraintTeacherSetNotAvailableTimesTest
-    {
-        Mock<DataModel> test;
-
-        [SetUp]
-        public void Init()
-        {
-            var data = new List<Tt_TimeOff>{
-                new Tt_TimeOff{ItemId = 4, day = 2, lessonIndex = 3, ItemType = 1},
-            }.AsQueryable();
-
-            var mockSet = new Mock<DbSet<Tt_TimeOff>>();
-            mockSet.As<IQueryable<Tt_TimeOff>>().Setup(m => m.Provider).Returns(data.Provider);
-            mockSet.As<IQueryable<Tt_TimeOff>>().Setup(m => m.Expression).Returns(data.Expression);
-            mockSet.As<IQueryable<Tt_TimeOff>>().Setup(m => m.ElementType).Returns(data.ElementType);
-            mockSet.As<IQueryable<Tt_TimeOff>>().Setup(m => m.GetEnumerator()).Returns(data.GetEnumerator());
-
-            var dataEmp = new List<HR_MasterData_Employees>{
+﻿using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Xml.Linq;
+using Timetabling.DB;
+
+namespace Timetabling.Objects.Constraints.TimeConstraints.Tests
+{
+    [TestFixture()]
+    public class ConstraintTeacherSetNotAvailableTimesTest
```

Comment:
Why the change from internal to public here? Class should not be visible outside the testing library.

Reply:
It should be possible to change it, I will need it in an other part.